internal/tfimport/tfimport_test.go

```diff
@@ -201,3 +202,816 @@ func TestDoesNotExist(t *testing.T) {
 		}
 	}
 }
+
+// Simple sanity tests, they make sure each importer assembles the plan values correctly for the ImportID.
+func TestImportersSanity(t *testing.T) {
+	tests := []struct {
+		resource   string
+		planFields map[string]interface{}
+		want       string
+	}{
+		{
+			"google_app_engine_application",
+			map[string]interface{}{
+				"project": "my-appengine-project",
+			},
+			"my-appengine-project",
+		},
+		{
+			"google_bigquery_dataset",
+			map[string]interface{}{
+				"project":    "my-bq-project",
```

Comment:
Do you currently have a test for the ImportID that if one of the fields it's looking for is missing, then it will error out?

Reply:
I don't, I'll add one, good idea.